containers/ddev-router/Dockerfile

```diff
@@ -6,22 +6,60 @@ ENV DEBIAN_FRONTEND noninteractive
 ENV DOCKER_GEN_VERSION 0.7.4
 ENV DOCKER_HOST unix:///tmp/docker.sock
 
+# TARGETPLATFORM is Docker buildx's target platform (e.g. linux/arm64), while 
+# BUILDPLATFORM is the platform of the build host (e.g. linux/amd64)
+ARG TARGETPLATFORM
+ARG BUILDPLATFORM
+
+SHELL ["/bin/bash", "-c"]
+
+RUN mkdir /tmp/ddev && \
+    if [[ $TARGETPLATFORM == "linux/arm64" ]]; then \
+        printf '# jwilder/forego doesnt have an arm64 download so we resort to the original forego repo \n\
+FORGO_IS_TGZ="true" \n\
+FORGO_FILENAME="forego-stable-linux-arm64.tgz" \n\
+FORGO_URL="https://bin.equinox.io/c/ekMN3bCZFUn/forego-stable-linux-arm64.tgz" \n\
+# jwilder/docker-gen doesnt have an arm64 download so we created a custom build \n\
+DOCKER_GEN_URL="https://github.com/dennisameling/docker-gen/releases/download/0.7.5/docker-gen-linux-arm64-0.7.5.tar.gz" \n\
+DOCKER_GEN_FILE="docker-gen-linux-arm64-0.7.5.tar.gz" \n\
+# PR for official arm64 support pending: https://github.com/FiloSottile/mkcert/pull/284 \n\
+MKCERT_URL="https://github.com/dennisameling/mkcert/releases/download/${MKCERT_VERSION}-ddev/mkcert-${MKCERT_VERSION}-linux-arm64"' > /tmp/ddev/vars; \
```

Comment:
Shall we fork mkcert or do we wait for my PR in their repo to be merged?

Reply:
I PM'd FiloSottile on Twitter the other day about this. Got no response. I was asking him if he'd be open to adding a maintainer for mkcert. I think he's quite a bigwig on golang at google, probably way too busy. But no response at all.

I think we have to fork. I'll fork in drud's namespace and give you privileges. 

The other option is just to maintain the builds we need. Maybe that's more sustainable? I'm not sure exactly where to put them. A fork is such a convenient place to put them.

Reply:
I forked and made you maintainer. https://github.com/drud/mkcert

Reply:
BOOM and we have a release: https://github.com/drud/mkcert/releases/tag/v1.4.4

Instructions for creating a new release are here: https://github.com/drud/mkcert#creating-a-new-release